templates/element/routes_panel.php

```diff
@@ -3,13 +3,46 @@
  * @var \Cake\Routing\Route\Route[] $routes
  * @var string $matchedRoute
  */
+
+use Cake\Core\Plugin as CorePlugin;
 use Cake\Utility\Hash;
 
 $routes = Cake\Routing\Router::routes();
+
+$amount_of_routes_per_group = [];
```

Comment:
We conventionally use `$camelBacked` var names.

Reply:
converted that now 👍🏻